src/content/self-hosted/install/releases.mdx

```diff
@@ -7,14 +7,22 @@ id: releases
 
 ## 1.15.0
 
-TBD
+22 November 2023
 
 This version is compatible with Kubernetes versions 1.25 to 1.27.
 
 ### Improvements
 
 - Upgrade ingress-nginx to 1.9.4 (Helm chart version 4.8.3)  <!-- 7386 -->
 - Enabled [chroot mode](https://kubernetes.io/blog/2022/04/28/ingress-nginx-1-2-0/) by default in both ingress-nginx controllers managed by Okteto. More info in [Upgrade your Okteto instance section](https://www.okteto.com/docs/self-hosted/install/upgrade/#upgrading-to-okteto-115x) <!-- 7425 -->
+- `okteto up` sessions are now garbage collected if no files have been synced for an hour   <!-- 7424 -->
+- Upgrade nginx to [1.8.4](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.9.4) (chart 4.8.3) <!-- 7386 -->
+
+### Bug Fixes
+
+- Fix cli authentication when using openid "audience" or "resource" attributes  <!-- 7363 -->
```

Comment:
This one was already mentioned on `1.14.1` release notes. I think we don't usually include it in the next minor.

If so, we should also mention the fix for the support of registries with custom ports